CleverTap/Plugins/iOS/CleverTapUnityManager.mm

```diff
@@ -74,8 +74,17 @@ + (void)launchWithAccountID:(NSString*)accountID andToken:(NSString *)token {
     [self launchWithAccountID:accountID token:token region:nil];
 }
 
-+ (void)launchWithAccountID:(NSString*)accountID token:(NSString *)token region:(NSString *)region {
++ (void)launchWithAccountID:(NSString *)accountID token:(NSString *)token region:(NSString *)region {
     [CleverTap setCredentialsWithAccountID:accountID token:token region:region];
+    [self performCommonLaunchTasks];
+}
+
++ (void)launchWithAccountID:(NSString *)accountID token:(NSString *)token proxyDomain:(NSString *)proxyDomain spikyProxyDomain:(NSString *)spikyProxyDomain {
+    [CleverTap setCredentialsWithAccountID:accountID token:token proxyDomain:proxyDomain spikyProxyDomain:spikyProxyDomain];
+    [self performCommonLaunchTasks];
+}
+
++ (void)performCommonLaunchTasks {
```

Comment:
Please rename this method. Use launch or something descriptive.

Reply:
Renamed it to initializeAndNotifyLaunch